ui_theme.py

```
@@ -0,0 +1,655 @@
+            try:
+                apps_use_light, _ = winreg.QueryValueEx(key, "AppsUseLightTheme")
+            finally:
+                winreg.CloseKey(key)
+            return not bool(apps_use_light)
+        except (OSError, FileNotFoundError, ImportError):
+            return None
+    if sys.platform == "darwin":
+        try:
+            import subprocess
+            r = subprocess.run(
+                ["defaults", "read", "-g", "AppleInterfaceStyle"],
```

Ruff (0.15.10)

[error] Starting a process with a partial executable path

(S607)